src/app/team-membership-dialog/team-membership-dialog.component.less

```diff
@@ -1,11 +1,11 @@
-@import '../../assets/stylesheets/base';
+@import (reference) '../../assets/stylesheets/_base';
 .createTeamMembershipBlock {
   font-size: rem(10);
   padding: 0;
   >div {
     margin-top: em(15);
     label {
-      width: $width100;
+      .width-100;
```

Comment:
I believe this should just be 100%. The width-100 class is really only used a shortcut used in the HTML pages. 

Due to the width-100 name, it's not likely to be a variable that will ever change from 100%.